From b4babae93b81be46c105e59c74b1708f00b68704 Mon Sep 17 00:00:00 2001
From: Carmine Scarpitta <cscarpit@cisco.com>
Date: Sat, 23 Nov 2024 08:46:00 +0100
Subject: [PATCH 4/5] tools: Add missing keyword `segment-routing srv6` in
 frr-reload

Fix the error below:

```
[58564|mgmtd] sending configuration
[58565|zebra] sending configuration,
line 10: % Unknown command[52]:  node-msd
[58573|isisd] sending configuration
[58565|zebra] Configuration file[/etc/frr/frr.conf] processing failure: 2
```

Signed-off-by: Carmine Scarpitta <cscarpit@cisco.com>
---
 tools/frr-reload.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tools/frr-reload.py b/tools/frr-reload.py
index 08a1f1e07e..baa6285f45 100755
--- a/tools/frr-reload.py
+++ b/tools/frr-reload.py
@@ -255,7 +255,9 @@ ctx_keywords = {
     },
     "router rip": {},
     "router ripng": {},
-    "router isis ": {},
+    "router isis ": {
+        "segment-routing srv6": {},
+    },
     "router openfabric ": {},
     "router ospf": {},
     "router ospf6": {},
-- 
2.39.5

